### ISSUE TITLE
Localizable URLs

### ISSUE THREAD

**Issue by hishamco** [simplcommerce/SimplCommerce#591]
URL localization increases the chances of matching search results in other cultures/languages

**Comment by thiennn** [comment]
This should be done after content localization (product, categories ...) is implemented.

**Comment by Mirivhbv** [comment]
Does anyone work on this issue? I consider product details, category name localization and etc.

**Comment by hishamco** [comment]
Nope, but it is in my roadmap, but I need @thiennn clarify which content to be localized, and open another issue with that
This is issue is specific for localizing URLs
